plugins/extractit/plugin.spec.yaml

```diff
@@ -4,10 +4,10 @@ products: ["insightconnect"]
 name: extractit
 title: ExtractIt
 description: The ExtractIt plugin has a collection of actions used to extract various information from text. These include URLs, domains, emails, IPs, and more
-version: 2.3.1
+version: 3.0.0
```

Comment:
This should be a minor version bump since the inputs added are all optional.

Reply:
Was under the impression that an empty string was required at the end of all enums in plugin.spec.yaml. Just want to check if this is the case? New inputs for date_format were expected to be mandatory.

Reply:
Yep, that is correct, an empty string is needed. However, in looking at the inputs you added (https://github.com/rapid7/insightconnect-plugins/pull/1062/files#diff-a9ef80c2a1138eec88b2f099658978e5d5a815265712c071c1f3702a50fca8a0R254, for example), it is marked `required: false`, which means it is optional and therefore a major version bump will not be required.

Reply:
Updated to `required: true` as this is a mandatory field